pkg/utils/logs/logs.go

```diff
@@ -80,3 +81,24 @@ func InitLogs(LogFlushFrequency time.Duration) {
 func FlushLogs() {
 	glog.Flush()
 }
+
+//timeNow: Current system time, accurate to microseconds, for example:
+//timeNow := time.Now().UTC().Format("2006-01-02 15:04:05.000000")
+//LogPrefix: multi-cloud、hotpot、nbp
+//os.Getpid: current process id
+//logLevel: Log level,such as INFO、WARNING、ERROR、FATAL.
+//filename: go file name.
+//line: Line Number.
+//funcname: function name
+//logMsg: log detail Messages
+//errorCode: if logLevel is ERROR ,else errorCode=""
+func GetLogContent(logPrefix string, pid string, logLevel string, fileName string, line string, funcName string, logMsg string, errorCode string) string {
```

Comment:
Why do we need to manually give pid, filename, line, funcname etc...? Please avoid that from the framework call. It can be automatically taken by the framework right? Please check how to optimize the same.
Why don't we use some standard go log packages and update as needed? (log, logrus, glog ..?)

Reply:
+1 to using a standard logger...i recommend using [https://github.com/uber-go/zap](url)

Reply:
logrus is also one of the widely used package. It provides capability and flexibility which can be quite useful.